TestModels/SimpleTypes/SimpleEnum/runtimes/rust/tests/simple_enum_test.rs

```diff
@@ -0,0 +1,51 @@
+use simple_enum::types::simple_enum_shape::SimpleEnumShape::*;
+use simple_enum::*;
+
+#[tokio::test]
+async fn test_get_enum() {
+    let result = client().get_enum().value(SECOND).send().await;
+    let output = result.unwrap();
+    let value = output.value().unwrap();
+    assert_eq!(value, SECOND);
```

Comment:
not sure how paranoid we want to be, but this could pass if `value` is not `SECOND` based on the `PartialEq` impl for `SimpleEnumShape`. it would need to be bizarrely horrible, but it's possible.

do we want to use `match` to assert it's the expected variant?

Reply:
I think I'm willing to trust  `#[derive(PartialEq)]` in this case.